### PR TITLE
Bump vscode engine to 1.81.1

### PR DESCRIPTION
**Pull request by Ao-senXiong**
Based on this issue https://github.com/dependabot/dependabot-core/issues/1655, it seems there is no update engine field in dependabot.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,4 +100,4 @@
   "lint-staged": {
     "**/*": "prettier --write --ignore-unknown src/*.js"
   }
-}
+}
```
Comment (wmdietl) [inline package.json:103]:
```suggestion
}
```

**Review by wmdietl — APPROVED** [review]
Thanks!
Is there a different dependency bot that handles this better?

File: package-lock.json [hunk 2]
```diff
@@ -21,9 +21,6 @@
         "lint-staged": "^14.0.1",
         "prettier": "3.0.2",
         "typescript": "^5.0.4"
-      },
-      "engines": {
```
Comment (wmdietl) [inline package-lock.json:25]: This removes the `engines` section from this file. Is this a manual edit? Or why is this removed now?

Comment (Ao-senXiong) [reply package-lock.json:25]: Sorry, this was a mistaken commit.

> This removes the `engines` section from this file. Is this a manual edit? Or why is this removed now?

**Comment by Ao-senXiong** [comment]
> Thanks! Is there a different dependency bot that handles this better?

People suggest GreenKeeper, but let me look around and I will make another PR if it works.

**Review by wmdietl — APPROVED** [review]
Thanks!
